CONTRIBUTING.md

```diff
@@ -3,18 +3,21 @@
 Thank you for your interest in contributing to Openverse! This document is
 a set of guidelines to help you contribute to this project.
 
+Please refer to [CONTRIBUTING.md](https://github.com/WordPress/openverse/blob/0b5e24bd325da7b75db9484a268d7c8d197ae920/CONTRIBUTING.md) for a
```

Comment:
We should update the link so that it always points to the latest.

```suggestion
Please refer to [CONTRIBUTING.md](./CONTRIBUTING.md) for a
```

Reply:
This would point to the file in the `openverse-catalog` repo, right? The requirement was to direct users to the main repo, so I put up the entire link.

Reply:
@dhruvkb we intentionally wanted to point to the larger contributing file in the main repo!